include/seqan3/search/fm_index/fm_index_cursor.hpp

```diff
@@ -106,15 +108,18 @@ class fm_index_cursor
     }
 
     //!\brief Optimized backward search without alphabet mapping
-    template <detail::sdsl_index csa_t>
-    bool backward_search(csa_t const & csa, sdsl_char_type const c, size_type & l, size_type & r) const noexcept
+    template <detail::sdsl_index sdsl_index_type>
```

Comment:
when you use the class member type within the function, you do not need the template at all :) you can remove this line

Reply:
Yes, that did the trick. Thx!